settings.gradle.kts

```diff
@@ -1,7 +1,5 @@
 include(":app")
 
-rootProject.name = "Jellyfin"
```

Comment:
Why remove that? :thinking:

Reply:
It doesn't do anything except telling IntelliJ/Android Studio to name the project "Jellyfin" in all places (which is annoying when I have multiple Jellyfin projects, just use the directory name please)

Reply:
Fair enough, we could change that to Jellyfin Android though.